src/m365/spo/commands/listitem/listitem-get.ts

```diff
@@ -143,14 +151,34 @@ class SpoListItemGetCommand extends SpoCommand {
     };
 
     try {
-      const response = await request.get<any>(requestOptions);
-      delete response['ID'];
-      logger.log(<ListItemInstance>response);
+      const itemProperties = await request.get<any>(requestOptions);
+      if (args.options.withPermissions) {
+        requestOptions.url = `${requestUrl}/items(${args.options.id})/RoleAssignments?$expand=Member,RoleDefinitionBindings`;
+        const response = await request.get<{ value: any[] }>(requestOptions);
+        const roleAssignments = this.setFriendlyPermissions(response.value);
+        itemProperties.RoleAssignments = roleAssignments;
+      }
+      delete itemProperties['ID'];
+      logger.log(<ListItemInstance>itemProperties);
```

Comment:
shouldn't we extend the `ListItemInstance` interface with `RoleAssignments` property to make it clean?